dist/tools/backport_pr/backport_pr.py

```diff
@@ -135,6 +135,16 @@ def main():
     if status != 200:
         print("Could not retrieve user: {}".format(user['message']))
         exit(1)
+    scopes = dict(g.getheaders())
+    if 'X-OAuth-Scopes' in scopes:
+        scopes = scopes['X-OAuth-Scopes']
+    else:
+        scopes = scopes['x-oauth-scopes']
+    scopes = scopes.split(',')
+    scopes = [x.strip() for x in scopes]
```

Comment:
Nitpick, feel free to disregard if you don't agree
```suggestion
    scopes = [x.strip() for x in scopes.split(',')]
```

Additionally I'd really prefer it if you don't reuse the variable names here for different representation of the same data.

Reply:
k